package.json

```diff
@@ -93,6 +93,7 @@
     "metalsmith": "^2.5.0",
     "mocha": "10.2.0",
     "ol-mapbox-style": "^11.0.1",
+    "ol-stac": "^1.0.0-beta.5",
```

Comment:
Adding this to our dev dependencies will make it available when building the examples.  However, we'll likely get into a situation with duplicate version of `ol` since [your package lists `ol`](https://github.com/m-mohr/ol-stac/blob/v1.0.0-beta.5/package.json#L31) as a direct dependency.

When we release 8.0.0, for example, since that version doesn't satisfy your `^7.0.0` constraint, there will be multiple versions of the `ol` package installed and bundled with the examples.

I know that package versioning can be a mess, but my preference when publishing a "plugin" type package is to _not_ include it in `dependencies`.  Instead, I think it works ok if you have a very generous version constraint listed in `peerDependencies` and then you can have a more specific/constrained version in `devDependencies`.

As an example, here is a popular looking Vue component library that doesn't list `vue` in its `dependencies` but does list it in its `devDependencies`: https://github.com/vuetifyjs/vuetify/blob/v3.3.14/package.json

And a similarly popular React component library that doesn't list `react` in its `dependencies` but does list it in its `peerDependencies`: https://github.com/mui/material-ui/blob/v5.14.6/packages/mui-base/package.json

Reply:
Thanks, this makes sense, I just need to check whether that has any side effects.

Reply:
Done, seems to work just fine.